polybar-scripts/info-healthchecks/info-healthchecks.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Your API Key
+API_KEY="pPi37pxsjljbQRDjYoKkFtwrRikrdnZy"
```

Comment:
You key is compromised now.

Reply:
Rookie mistake, was using a test account though.